uid2_client/uid2_token_generator.py

```diff
@@ -98,13 +99,15 @@ def generate_uid2_token_v4(id_str, master_key, site_id, site_key, params=default
 
     @staticmethod
     def generate_uid_token(id_str, master_key, site_id, site_key, identity_scope, token_version,
-                           created_at=None, expires_at=None):
+                           identity_established=None, token_generated=None, token_expiry=None):
```

Comment:
Just curious, why the default value is not `datetime.now()` but `None`

Reply:
@caroline-ttd  [Default values are only evaluated once](https://docs.python.org/3/tutorial/controlflow.html#default-argument-values).